Jenkinsfile

```diff
@@ -1,17 +1,24 @@
 pipeline {
-  agent any
+  agent none
   stages {
-    stage('') {
-      agent {
-        node {
-          label 'linux'
+    stage('test') {
+      parallel {
+        stage('python3') {
+          agent {
+            dockerfile {
+              // Large image with full OpenModelica build dependencies; lacks omc and OMPython
+              label 'linux'
+              dir '.jenkins/python3'
+              additionalBuildArgs '--pull'
+            }
+          }
+          steps {
+            sh 'hostname'
+            sh 'HOME="$PWD" python3 setup.py install --user'
+            sh 'pytest testing'
```

Comment:
Don't you also need HOME="$PWD" for the pytest call?

Reply:
Seems so

Reply:
Wonder if pytest can find libs installed using `--user`.